RepoDb.Core/RepoDb/Reflection/Compiler/Compiler.cs

```diff
@@ -1547,6 +1547,17 @@ internal static Expression ConvertExpressionToClassHandlerSetExpression<TResult>
             return Expression.Call(parameterVariableExpression, GetDbParameterValueSetMethod(), expression);
         }
 
+        private static DbType? GetDbType(ClassProperty classProperty, Type dbFieldType)
```

Comment:
As part of our [coding standard](https://github.com/mikependon/RepoDB/blob/master/RepoDb.Docs/coding-standards.md#arguments), you might notice that we are always doing a "new line" per argument.

```csharp
private static DbType? GetDbType(ClassProperty classProperty,
      Type dbFieldType)
{
     ...
}
```

But I am happy that you are very engage here, I know that next time, you will be one of my biggest contributor and/or active collaborator. Thanks as always 😺